src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms/Service/NavigationService.show.cs

```diff
@@ -40,11 +39,11 @@ public virtual async Task ShowAsync<TViewModel>()
                 page);
         }
 
-        /// <inheritdoc cref="CodeMonkeys.Core.Interfaces.Navigation.IViewModelNavigationService.ShowAsync{TViewModelInterface, TModel}(TModel)" />
-        public virtual async Task ShowAsync<TViewModel, TModel>(
-            TModel model)
+        /// <inheritdoc cref="CodeMonkeys.Navigation.INavigationService.ShowAsync{TViewModelInterface, TModel}(TModel)" />
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 122 characters long line (which is greater than 120 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5598919)